recipes/classix/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: classix
+  version: 0.1.1
+
+
+source:
+  fn: ClassixClustering-0.5.8.tar.gz
```

Comment:
You don't need to include the `fn` line.

Reply:
Thaniks, resolve it!